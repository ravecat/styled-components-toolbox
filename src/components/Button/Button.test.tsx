import React from "react";
import renderer from "react-test-renderer";

import { Button } from "./index";

it("<Button> renders correctly", () => {
  const tree = renderer.create(<Button>Button</Button>).toJSON();
  expect(tree).toMatchSnapshot();
});
