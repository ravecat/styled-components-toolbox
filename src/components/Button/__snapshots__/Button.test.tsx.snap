// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button> renders correctly 1`] = `
<button
  className="sc-bdfBwQ sc-gsTCUz goIjVs csthOT"
  type="button"
>
  Button
</button>
`;
